.github/workflows/notion.yml

```
@@ -0,0 +1,101 @@
+          PBI_NUMBER=$(echo "${{ github.event.pull_request.title }}" | grep -oE 'PBI [0-9]+' | awk '{print $2}')
+          if [ -z "$PBI_NUMBER" ]; then
+            echo "PBI number not found in PR title, skipping Notion update"
+            echo "HAS_PBI=false" >> $GITHUB_ENV
+          else
+            echo "PBI_NUMBER=$PBI_NUMBER" >> $GITHUB_ENV
+            echo "HAS_PBI=true" >> $GITHUB_ENV
+          fi
+
+      - name: Post warning comment for missing PBI
+        if: env.HAS_PBI == 'false'
+        uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)